rasa/cli/x.py

```diff
@@ -111,11 +111,28 @@ def _overwrite_endpoints_for_local_x(
     from rasa.utils.endpoints import EndpointConfig
     import questionary
 
-    endpoints.model = EndpointConfig(
-        "{}/projects/default/models/tags/production".format(rasa_x_url),
-        token=rasa_x_token,
-        wait_time_between_pulls=2,
-    )
+    # Checking if endpoint.yml has existing url and wait time values set, if so give warning we are overwriting
+    # the endpoint.yml file.
+    custom_wait_time_pulls = endpoints.model.kwargs["wait_time_between_pulls"]
+    custom_url = endpoints.model.url
+
+    if custom_url is not None:
+        cli_utils.print_warning(
```

Comment:
let's use `logger` with level `debug` or `info`.
Regarding the message: I'd frame it a bit differently e.g. `Ignoring url '{custom_url}' from 'endpoints.yml' and using æ{model_pull_url}' instead.`

Reply:
for my own understanding why do we have this cli_utils.print options if we are just using logger or what do we get by using one over the other?

Reply:
Updated the logging section based on your feedback, I made it info for now which only shows up in --debug mode anyways.